admin-console/src/main/webapp/src/components/CreateEditModal.tsx

```diff
@@ -72,9 +70,10 @@ export const CreateEditModal: React.FunctionComponent<CreateEditModalProps> = (p
                             name="description"
                         /></FormGroup>
                     <ActionGroup>
-                        <Button variant='primary' type='submit'
-                            { ...props.isLoading ? <Spinner /> : props.eventTypeQuery }
```

Comment:
This syntax is a bit strange, spreading a component inside the properties?  anyhow updated below